FileUploadApi.md

```diff
@@ -0,0 +1,263 @@
+# File Upload API
+
+## Introduction
+
+This API is responsible for uploading files to storage. 
+mod-data-import module use uploaded files for parsing and importing files to storage and another FOLIO modules usage.
+File Upload API provides a possibility to upload several files and store state of uploaded files. 
```

Comment:
ability, not a possibility
ability is something an object is able to do
possibility describes a future event. the likelihood something may happen